package.json

```diff
@@ -1,6 +1,7 @@
 {
-  "name": "root",
+  "name": "pid-component-monorepo",
   "private": true,
+  "version": "0.0.0",
```

Comment:
Is this the right way to do this? Seems strange. For the automatic KPI generation (with Zenodo) the version has to be accurate.